macrosynergy/pnl/naive_pnl.py

```diff
@@ -61,16 +62,17 @@ def __init__(
         end: str = None,
         blacklist: dict = None,
     ):
-        df["real_date"] = pd.to_datetime(df["real_date"], format="%Y-%m-%d")
+        cols = ["cid", "xcat", "real_date", "value"]
 
+        df = QuantamentalDataFrame(df[cols])
```

Comment:
I think since this is only used once, just make this:
`df = QuantamentalDataFrame(df["cid", "xcat", "real_date", "value"])`